Formula/iojs.rb

```diff
@@ -33,15 +33,20 @@ def install
       ENV["NPM_CONFIG_LOGLEVEL"] = "verbose"
 
       cd buildpath/"npm_install" do
-        # Patch node-gyp until github.com/TooTallNate/node-gyp/pull/564 is resolved
-        # Patch extracted from https://github.com/iojs/io.js/commit/82227f3
-        p = Patch.create(:p1, :DATA)
-        p.path = Pathname.new(__FILE__).expand_path
-        p.apply
         system "./configure", "--prefix=#{libexec}/npm"
         system "make", "install"
       end
 
+      # Pre-download the sources for node-gyp until github.com/TooTallNate/node-gyp/pull/564 is resolved
+      if File.directory?("#{ENV['HOME']}/.node-gyp/#{version}")
+        system "rm", "-rf", "#{ENV['HOME']}/.node-gyp/#{version}"
```

Comment:
You wouldn't need to lean on the system here. Ruby's FileUtils can handle a lot of this. ` rm_rf "#{ENV["HOME"]}/.node-gyp/#{version}" ` should work, for example.